README.md

```diff
@@ -50,6 +50,7 @@ Usage
             ],
             "recurse": true,
             "replace": false,
+            "skip": false,
```

Comment:
I think calling this option `ignore-duplicates` might be a bit more intuitive than `skip`. There should probably also be documentation that points out that this feature flag and `replace` flag are mutually exclusive. It actually might be a good idea to add a check when reading the configuration and at least log a warning if both `replace` and `ignore-duplicates` are provided and true.

I worry a bit that we are growing the set of feature flags to the point that it will be hard for anyone to reason about which flags to set for a given project, but this one additional flag is probably not the breaking point for the current system. A reasonable 2.0 breaking API change would be to combine the `replace` and `ignore-duplicates` flags into a new `merge-strategy` setting that would encompass the `resolve`, `replace`, and `ignore` behaviors.